core/biogeophys/cable_common.F90

```diff
@@ -684,6 +693,136 @@ subroutine get_unit(iunit)
 
   end subroutine get_unit
 
+  FUNCTION get_dimid(FileID, DimNames)
+    !*## Purpose
+    !
+    ! Find the id of the desired dimension using a list of possible names.
+    !
+    !##
+    !
+    ! Run through a list of possible names for the dimensions as defined in the
+    ! cable_common module.
+
+    ! Input and output
+    INTEGER :: FileID, get_dimid
+    CHARACTER(LEN=*), DIMENSION(:)  :: DimNames
+
+    ! Iterator and status checker- use NF90_EDIMMETA so we throw error when an
+    ! empty list of names is passed.
+    INTEGER :: i, ok = NF90_EBADDIM
+
+    ! Run through the possible names until we find the correct one
+    CheckNames: DO i = 1, SIZE(DimNames)
+      ok = NF90_INQ_DIMID(FileID, TRIM(DimNames(i)), get_dimid)
+      IF (ok == NF90_NOERR) THEN
+        EXIT CheckNames
+      END IF
+    END DO CheckNames
+
+    CALL handle_err(ok, "Failed to find any "//TRIM(DimNames(1))//" dimensions&
+      from the list.")
+
+  END FUNCTION get_dimid
+
+  !FUNCTION latitude_dimid(FileID)
+    !!*## Purpose
```

Comment:
Not sure why we have commented these out (we can delete these right)?

Reply:
Oh yep, they will be removed.